app/src/androidTest/java/com/android/unio/end2end/EventCreationE2ETest.kt

```diff
@@ -275,7 +275,7 @@ class EventCreationE2ETest : EndToEndTest() {
     composeTestRule.waitUntilExactlyOneExists(
         matcher =
             hasTestTag(EventCreationTestTags.LOCATION_SUGGESTION_ITEM_LATITUDE + EVENT_LATITUDE),
-        timeoutMillis = 10000)
+        timeoutMillis = 15000)
```

Comment:
Waiting 15 seconds befor etimeout is a lot. Confirm me you really need to wait that long, otherwise, reduce the sleeping time

Reply:
Nice spot, I had a problem with the test and wanted to see if it was due to insufficient delay, but I forgot to reset the correct timeout